### PR TITLE
Add contributing guidelines

### PR DESCRIPTION
**Pull request by ClemensLinnhoff**
Related to #14

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -116,11 +116,12 @@ In order to create a branch, either fork the repository or request access to the
 Be sure to link the issue you created earlier in the pull request.
 - In the bottom, you can select to either submit as a pull request or as a draft pull request.
 - Use a draft pull request first, so the community can already see the ongoing work.
+- In this stage, the review from parson takes place.
```
Comment (ClemensLinnhoff) [inline CONTRIBUTING.md:119]: Let's not name the service provider in the contribution guideline. It should be neutral and valid after the duration of this project.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -116,11 +116,12 @@ In order to create a branch, either fork the repository or request access to the
...
   
 Also, the CI pipeline will run for every commit, so you can continuously check your work.
 
 ### Convert into 'real' pull request
 
-Once you are done with your changes, [convert the draft to a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request).
+Once you are done with your changes and the review from parson took place, [convert the draft to a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request).
```
Comment (ClemensLinnhoff) [inline CONTRIBUTING.md:125]: see above

File: .antora/antora.yml [hunk 3]
```diff
@@ -21,7 +21,7 @@ asciidoc: # Sets global AsciiDoc attributes that are applied for every page of t
     # START - Mandatory ASAM attributes
     use-antora-rules: true
     THIS_STANDARD: ASAM OpenMATERIAL<sup>®</sup> # Normal attribute
-    THIS_STANDARD_TITLE: 'ASAM OSI^®^' # Normal attribute
+    THIS_STANDARD_TITLE: 'ASAM OpenMATERIAL^®^' # Normal attribute
```
Comment (ClemensLinnhoff) [inline .antora/antora.yml:24]: This has nothing to do with this PR, please revert and fix on a different branch @FKlopfer.

File: .vscode/settings.json
Comment (ClemensLinnhoff) [inline .vscode/settings.json:1]: This has nothing to do with this PR, please revert and fix on a different branch @FKlopfer.

File: CONTRIBUTING.md (hunk 1 above)
Comment (FKlopfer) [reply CONTRIBUTING.md]: Agreed. Changed it to "service provider".

File: CONTRIBUTING.md (hunk 2 above)
Comment (FKlopfer) [reply CONTRIBUTING.md]: Agreed. Changed it to "service provider".

File: .antora/antora.yml (hunk 3 above)
Comment (FKlopfer) [reply .antora/antora.yml]: Revert changes. Will fix it on another branch.

File: .vscode/settings.json
Comment (FKlopfer) [reply .vscode/settings.json:1]: Revert changes. Will fix it on another branch.

**Comment by ClemensLinnhoff** [comment]
CCB decision 26.06.24: PR will be closed as contribution guidelines will be handled in the ASAM organization.

**Pull request closed without merging**